clients/drcachesim/common/trace_entry.h

```diff
@@ -226,6 +226,15 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_FUNC_RETVAL,
 
+    /* This is a non-public type only present in an offline raw trace.  To support a
```

Comment:
nit: the rest of the file has single spaces after punctuation.

Reply:
Fixed.